backend/views/partials/errorCard.ejs

```diff
@@ -0,0 +1,4 @@
+<div class="errors-card" id="<%= cardId %>" style="display: <%- display%>">
```

Comment:
Because the same error message card is used twice, I created a partial for it. I think this is where it is supposed to go in the file structure but definitely let me know if its wrong